### PR TITLE
Style Changes

### PR DESCRIPTION
**Pull request by mathurn**
I'm a noob at JS/React so not fully understanding some of this stuff. Creating a PR for comments about some of these changes. Also, we should clean up the redundant `src/` and have a `config/` directory for `firestoreConfig.js`.

### CODE REVIEW

File: app/src/AddWordsComponents.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ class AddWordsComponent extends React.Component {
 		console.log(values);
 		const db = firebase.firestore();
 		const room = "someRoom"; // Replace this with getting dynamic room ID
-		var onSubmit = this.props.onSubmit;
+		const onSubmit = this.props.onSubmit; // why var?
```
Comment (ennsharma) [inline app/src/AddWordsComponents.js]: +1, everything possible should be marked as `const` (which is most things, in my experience)

Comment (mathurn) [reply app/src/AddWordsComponents.js]: kk, will remove comment from this line

File: app/src/JoinRoomCard.js [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 import "./css/main.css";
+
 const firebase = require("firebase");
 const assert = require("assert")
 require("firebase/firestore");
 
 class JoinRoomCard extends React.Component {
   submitValues(values, setSubmitting) {
-    console.log("Joining Room:", values)
+    console.log("Joining Room: " + values);
```
Comment (ennsharma) [inline app/src/JoinRoomCard.js]: Actually JS is nicer than this, you can do 
```suggestion
    console.log(`Joining Room: ${values}`);
```
and we should adopt this for all embedded variables in strings (as well as `;` as convention after everything)

File: app/src/CreateRoomCard.js [hunk 3]
```diff
@@ -30,8 +31,7 @@ class CreateRoomCard extends React.Component {
           <Formik
             initialValues={{ 'userName': '', 'roomName': '', 'password': '' }}
             validate={values => {
-              const errors = {};
-              return errors;
+              return {};
```
Comment (ennsharma) [inline app/src/CreateRoomCard.js]: If we're just returning an object without doing any additional processing, you could directly do
```suggestion
values => {}
```

**Review by chriscorrea14 — CHANGES_REQUESTED** [review]
no

**Review by ennsharma — APPROVED** [review]
yes

File: app/src/JoinRoomCard.js [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
 import "./css/main.css";
+
 const firebase = require("firebase");
 const assert = require("assert")
 require("firebase/firestore");
 
 class JoinRoomCard extends React.Component {
   submitValues(values, setSubmitting) {
-    console.log("Joining Room:", values)
+    console.log(`Joining Room: ${values}`);
```
Comment (chriscorrea14) [inline app/src/JoinRoomCard.js:11]: wat dis

Comment (mathurn) [reply app/src/JoinRoomCard.js:11]: Apparently JS string interpolation [uses backtick](https://dmitripavlutin.com/string-interpolation-in-javascript/)

**Pull request merged**